dist/messages/messages.json

```diff
@@ -28,5 +28,6 @@
   "uploaded_parasoft_report_results": "Uploaded Parasoft {0} Static Analysis results: {1} vulnerabilities",
   "failed_to_create_report_module": "Failed to create Parasoft {0} report module. Error: {1}",
   "failed_to_upload_parasoft_report_results": "Failed to upload Parasoft {0} Static Analysis results. Error: {1}",
-  "mark_build_to_failed_due_to_vulnerability": "Mark the build as failed, due to vulnerabilities found in Parasoft Static Analysis reports"
+  "mark_build_to_failed_due_to_vulnerability": "Mark the build as failed, due to vulnerabilities found in Parasoft Static Analysis reports",
+  "flow_or_duplicate_violation_details_description": "This violation is a {0} type violation, you can see more details in report.html in the report generation folder"
```

Comment:
=> This is a {0} analysis violation. More details can be found in the HTML report generated by the Parasoft tool (C/C++test, Jtest, or dotTEST).